notebooks/02_numerical_pipeline_scaling.ipynb

```diff
@@ -540,14 +540,14 @@
     "\n",
     "<div class=\"admonition note alert alert-info\">\n",
     "<p class=\"first admonition-title\" style=\"font-weight: bold;\">Note</p>\n",
-    "<p class=\"last\">This figure shows the particular case of K-fold cross-validation strategy.\n",
+    "<p class=\"last\">This figure shows the particular case of <strong>K-fold</strong> cross-validation strategy.\n",
+    "For each cross-validation split, the procedure trains a model on all the red\n",
+    "samples and evaluate the score of the model on the blue samples.\n",
     "As mentioned earlier, there are a variety of different cross-validation\n",
-    "strategies. Some of these aspects will be covered in more details in future\n",
+    "strategies. Some of these aspects will be covered in more detail in future\n",
     "notebooks.</p>\n",
     "</div>\n",
     "\n",
-    "For each cross-validation split, the procedure trains a model on all the red\n",
-    "samples and evaluate the score of the model on the blue samples.\n",
     "Cross-validation is therefore computationally intensive because it requires\n",
     "training several models instead of one.\n",
     "\n",
```

Comment:
Are those changes already present in the matching python script? I am worried that the explanation about the meaning of the colors has disappeared from this `02_numerical_pipeline_scaling` notebook.

Reply:
In principle they are now inside the note. But I just noticed this will have some conflicts with #498.
We will have to synchronize the jupyter notebooks after solving the conflicts